html/pfappserver/root/static/admin/status.js

```diff
@@ -305,23 +305,21 @@ function initDashboard() {
     var cluster = JSON.parse(clusterEl.textContent || clusterEl.innerHTML);
 
     /* Hide missing charts */
-    validateCharts();
+    setInterval(validateCharts, 100);
```

Comment:
Can we find an alternative to calling this function 10 times per __second__?

Reply:
That would be the one I found, let me know if you have an alternative. From my testing calling this was very lightweight so 10 times per second didn't account to much CPU usage